cpr/session.cpp

```diff
@@ -38,6 +38,7 @@ class Session::Impl {
     void SetLowSpeed(const LowSpeed& low_speed);
     void SetVerbose(const Verbose& verbose);
     void SetVerifySsl(const VerifySsl& verify);
+    void SetInterface(const Interface& interface);
```

Comment:
"Interface" is pretty generic, how about "NetworkInterface" ?  Maybe that will fix the windows build failure as well?

Reply:
That sounds good. I will create another pull request with the change.